Example/ImportExample01.md

```diff
@@ -0,0 +1,104 @@
+## An example
+
+Imagine you have your own Frost-Server for sensor measurements. Unfortunately you have your old data stored in csv and excel files and now you want them online on your Frost-Server.
+Thats the perfect usecase for ChillImport. This software provides you an easy to use GUI to select and import your old sensor data.
+
+First download and install the program.
+
+The csv-file you want to import is: Examplefile.csv
+
+### Step 1 
+(Tip: Open the log. It might help)
+1.	select "File"
+2.	then select the "examplefile.csv" file 
+3.	Click "ok" (every time u choose a new source you have to click ok)
+
+### Step 2
+Here you have 2 Options:
+
+1.	Choose a configuration
+
+2.	Create a new configuration
+
+We want to focus on option 2
+
+### Step 3
+Tip: Open the preview (Click on "Toggle Preview")
+
+1.	At first you have to connect to your Frost-Server. For example: <http://chillimport-frost.docker01.justAnExample.de/v1.0>. \
+	Click "OK".
+   
+2.	Choose the delimiter of the columns. Here it is the semicolon.
+
+3.	In the first line of our file are no measurements.
+	So we want to skip the first line. => Number of header lines = 1
+   
+4.	We skip the mapping for this file. We don´t want to map any measurements.
+
+5.	Timezone: choose the timezone where the measurements were created  (eg. UTC+1).
+	* The date and time in our file is stored in the first column (column 0) => Column = 0
+	* The format is: dd.MM.yyyy HH:mm 
+	* Please click "Next".
+
+![](step03_01.png)
+	
+6.	Now we create a thing. The thing is our object where our measurements were created. It can be a weather-station, a smart home or something else. 
+	1.	Click "Create Thing"

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[list-item-content-indent] Don’t use mixed indentation for children, remove 1 space](https://app.codacy.com/gh/FraunhoferIOSB/ChillImport/pullRequest?prid=4872216)